### PR TITLE
login frontend

### LINKED ISSUE #63: Buttons for user pages too large

### PR DESCRIPTION
**Pull request by akipekoren**
Different user roles are handled with one login page. 

Some example credentials  :  

* client@test.com 
123456789

* employee@test.com
123456789

*farmer@test.com
123456789

### CODE REVIEW

File: client/src/ui-components/LoginFormComponent/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            Submit
+          </Button>
+          <Button
+            className="button-reset"
+            type="submit"
+            onClick={props.handleReset}
+            size="lg"
+          >
+            Reset
+          </Button>
```
Comment (MrGreensushi) [inline client/src/ui-components/LoginFormComponent/LoginForm.js]: Fo the Button use the UI-component Button, it already has its own CSS

**Review by MrGreensushi — CHANGES_REQUESTED** [review]
Your page is a bit different from UI-design:
image

The login form doesn't accept '@' as a character: big problem if the login is done via email

**Review by MarcoBramini — DISMISSED** [review]
This PR needs to be updated to stick to the new login architecture.
Look at the userloginpage, has everything needed for the login but is missing the UI (I'm talking about the page with just the buttons to try different users)

**Comment by akipekoren** [comment]
Screen Shot 2021-12-05 at 14 19 02

**Review by MrGreensushi — APPROVED** [review]

**Pull request merged**